pkg/api/pod/warnings.go

```diff
@@ -32,6 +33,11 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
+var (
+	// minimum memory value, reference: https://github.com/moby/moby/blob/cd08d377c5e0348984d1f46d196337d895ec47ad/daemon/daemon_unix.go#L67-L68
+	minMemory = resource.NewQuantity(6*1024*1024, resource.BinarySI) // 6Mi
```

Comment:
Not all container runtimes are limited minimum memory to 6Mi, and our warnings will be inaccurate if users use other container runtimes besides containerd.

Reply:
yes, at present, only contaienrd has set a 6Mi limit. Just like the existing warnings, it will set the memory to something like `4m`, `100m`, and give a warning to inform the user that it should be set to an integer.
https://github.com/kubernetes/kubernetes/blob/51ad0bbb7364f68a5772f5f4b6561e919e53ca77/pkg/api/pod/warnings.go#L240-L242

the current pr setting warning is only to inform the user:`supply a reasonable functional container`

Reply:
I agree with @HirazawaUi. I'm not sure we can add warnings based on containerd minimum settings. I think the warnings should live in containerd or other container runtime logs.